query.js

```diff
@@ -61,16 +62,8 @@ exports.init = function (sbot, config) {
 
     collectMetadata(content) {
       const metadata = {}
-      const ignored = [
-        'feedpurpose',
-        'subfeed',
-        'nonce',
-        'metafeed',
-        'tangles',
-        'type',
-      ]
-      for (const key of Object.keys(content)) {
-        if (ignored.includes(key)) continue
+      for (const key in content) {
+        if (NOT_METADATA.has(key)) continue
```

Comment:
I think your PR #89 is going to grow more, so we could probably take these changes which are unrelated (they're left over from the reserved-keywords PR) and make it its own PR.

Reply:
they are related. I had to add `recps` to this `ignored` list. So it was add recps as a string to this file, or swap this NOT_METADATA in.

I don't intend to change this one